frontend/public/components/utils/dropdown.jsx

```diff
@@ -114,25 +114,33 @@ class DropDownRow extends React.PureComponent {
   }
 
   render () {
-    const {itemKey, content, onclick, onBookmark, onUnBookmark, className, selected, hover, canFavorite, onFavorite, favoriteKey} = this.props;
+    const {itemKey, content, onclick, onBookmark, onUnBookmark, className, selected, hover, canFavorite, onFavorite, favoriteKey, namespace} = this.props;
+
+    let url;
+
+    if (namespace) {
+      url = (content === 'all namespaces')
+        ? '/overview/all-namespaces'
+        : `/overview/ns/${content}`;
+    }
 
     let prefix;
     if (onUnBookmark) {
-      prefix = <a className={classNames('bookmarker', {focus: selected, hover})} onClick={e => onUnBookmark(e, itemKey)}><i className="fa fa-minus-circle" /></a>;
+      prefix = <a href={url} className={classNames('bookmarker', {focus: selected, hover})} onClick={e => onUnBookmark(e, itemKey)}><i className="fa fa-minus-circle" /></a>;
```

Comment:
We don't want to make them real links. We should use something like `href="#"` and then call `e.preventDefault()`. This makes it keyboard focusable without changing the behavior.

Reply:
Another fix that might work is giving `tabIndex=0` since we're already handling keyboard events